### PR TITLE
Update changelog for v1.0.1

### PR DESCRIPTION
**Pull request by pinin4fjords**
Changelog for https://github.com/nf-core/differentialabundance/pull/49

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,12 +5,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## v1.0.0 - 2022-01-23
```
Comment (jfy133) [inline CHANGELOG.md:6]: Should be .1?

**Review by jfy133 — COMMENTED** [review]
From my phone

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,14 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.0.1 - 2022-01-24
```
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## v1.0.1 - 2022-01-25
```

**Review by jfy133 — APPROVED** [review]

**Comment by pinin4fjords** [comment]
Thanks @jfy133 !

**Pull request merged**